### DISCUSSION TITLE
How to use different kb profile for different kits?

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#49] [Q&A] [answered]
How to use different kb profile for different kits?
Using foxspigot

**Comment by HierarchyMan** [comment]
@GoodestEnglish

**Comment by GoodestEnglish** [comment, marked as answer]
It should set automatically if you have the knockback profile name set correctly. The knockback profile name should be the same as the kit name (not the display name, by what I mean name is the name you typed in `/kit create <kit name>`)

**Reply by HierarchyMan** [reply]
Else default spigot kb is applied?
And I donylt need eden api to get the kb right?

**Reply by GoodestEnglish** [reply]
The knockback will not be changed if the knockback profile isn't found
You don't need to use extra stuff to handle knockback, Eden will handle it by itself
